tests/test_loop.py

```diff
@@ -0,0 +1,20 @@
+import asyncio
+import platform
+import threading
+
+import pytest
+
+
+def skip_if_on_windows():
+    if platform.system() == "Windows":
+        pytest.skip("the test is not valid for Windows")
+
+
+@asyncio.coroutine
+def test_subprocess_co(loop):
+    skip_if_on_windows()
```

Comment:
Please replace it with `@pytest.skipif` decorator.
I pretty sure keeping a check alongside with test improves readability.